src/view/RecoExplorerApp.py

```diff
@@ -1370,6 +1378,7 @@ def append_block_to_navigation(self, block):
 
     #
     def assemble_components(self):
+        accordion_max_width = 323
```

Comment:
Wir sollten das zumindest in einer Konstanten auslagern.

Reply:
Ich habe es in ui_constants ausgelagert.
Mir scheint die Breite der sidebar auch nicht dynamisch zu sein, sondern fix. Deshalb finde ich kann man auch die Maxbreite hard kodieren.